src/lib389/lib389/cli_base/__init__.py

```diff
@@ -34,7 +33,7 @@ def _get_arg(args, msg=None, hidden=False, confirm=False):
                     x = getpass("%s : " % msg)
                     y = getpass("CONFIRM - %s : " % msg)
                     if x != y:
-                        log.info("Passwords do not match, try again.")
+                        print("Passwords do not match, try again.")
```

Comment:
Is it a typo? Other places (for both JSON and non-JSON) were changed to `log.info`.

Reply:
> Is it a typo? Other places (for both JSON and non-JSON) were changed to `log.info`.

We don't have "log" available.  Not sure why it previously worked, but it caused a lint error so I just cleaned it up

Reply:
Oh, ok, got it!